dolphinscheduler-api/src/test/java/org/apache/dolphinscheduler/api/service/ProjectServiceTest.java

```diff
@@ -159,31 +156,10 @@ public void testCheckProjectAndAuth() {
         result = projectService.checkProjectAndAuth(loginUser, project, projectCode, PROJECT);
         logger.info(result.toString());
         Assertions.assertEquals(Status.SUCCESS, result.get(Constants.STATUS));
-
```

Comment:
May I ask why do u remove these two cases?

Reply:
Hi, @EricGao888 , thanks for your comment. I removed these two cases because these two cases (PROJECT_NOT_EXIST and USER_NO_OPERATION_PROJECT_PERM) have been tested before, as shown in the figure below:

<img width="977" alt="截屏2023-02-23 09 57 18" src="https://user-images.githubusercontent.com/38122586/220805709-8c01cdd0-686c-48d9-8ef7-2bf38e62b60e.png">